core/main/src/processor/main_context_processor.rs

```diff
@@ -176,16 +175,8 @@ impl MainContextProcessor {
 
             if state.supports_app_catalog() {
                 let ignore_list = vec![state.get_device_manifest().applications.defaults.main];
-
```

Comment:
Got rid of `AppCatalogRequest::OnAppsUpdate` altogether, unnecessary request layer now that we've moved to OSS.